### PR TITLE
Fix error message showed when opening XLS file

### PR DESCRIPTION
**Pull request by MarwenBENALAYA**
I am targeting this branch, because {When downloading XLS file and opening it, a popup with message "File format and extension of 'filename.xls' don't match. The file could be corrupted or unsafe. Unless you trust its source, don't open it. Do you want to open it anyway?" showed. because the content file is HTML.}.

In case of bug fix, `1.x` **MUST** be targeted.

## Changelog

```markdown

### Changed
- Changing methods (XlsWriter::open(), XlsWriter::close(), XlsWriter::write($data), XlsWriter::init($data)) and using PHPExcel to cretae content of XLS file
```

## To do

- [ ] Update the tests

## Subject

- After downloading an XLS file, when I want to opening this file with Microsoft office Excel an message dialog showed "File format and extension of 'filename.xls' don't match. The file could be corrupted or unsafe. Unless you trust its source, don't open it. Do you want to open it anyway?" because the content of the file is HTML, I'm using a PHPExcel to generate a file

### CODE REVIEW

**Comment by greg0ire** [comment]
related : https://github.com/sonata-project/exporter/issues/37

**Comment by greg0ire** [comment]
@sonata-project/contributors if [this comment](https://github.com/sonata-project/exporter/issues/37#issuecomment-47359748) still holds true, I'm don't think we should merge this.

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "symfony/phpunit-bridge": "^3.3",
         "symfony/property-access": "^2.8 || ^3.2",
         "symfony/routing": "^2.8 || ^3.2"
```
Comment (core23) [inline composer.json]: There is a `,` missing at the end of the line

**Comment by OskarStark** [comment]
I am 👎 for merging this and prefer #128 

@MarwenBENALAYA can you take care of the #128 PR?

**Comment by MarwenBENALAYA** [comment]
@SonataCI 
I have created a new PR #189 containing the same work (rebased) as this PR.
You can close this PR.

**Comment by OskarStark** [comment]
@MarwenBENALAYA why did you do this? just `push --force` to this feature branch to update your PR.

I prefer to keep this PR instead of #189

**Comment by Soullivaneuh** [comment]
Close for #189.

Please next time, just update the PR.

**Comment by OskarStark** [comment]
@Soullivaneuh I already closed #189, but WDYT, IMO #128 is the correct solution to fix the problem

**Pull request closed without merging**